torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -642,8 +651,8 @@ void export_opnames(const script::Module& m, std::set<std::string>& opnames) {
     for (const auto& op : ops_list) {
       auto op_item = op.toTuple()->elements();
       TORCH_CHECK(
-          op_item.size() == 2,
-          "There should be two parts in an operator name.");
+          op_item.size() >= 2,
```

Comment:
is this because the third one can be num_speficied_args?

Reply:
Right. Before it has just name and overload. New the num_speficied_args is added.

Reply:
Same comment about documenting the "parts".